tests/bud.bats

```diff
@@ -693,3 +693,49 @@ load helpers
   buildah rm ${cid}
   buildah rmi ${target}
 }
+
+@test "bud-onbuild" {
+  target=onbuild
+  buildah bud --format docker --signature-policy ${TESTSDIR}/policy.json -t ${target} ${TESTSDIR}/bud/onbuild
+  run buildah --debug=false inspect --format '{{printf "%q" .Docker.Config.OnBuild}}' ${target}
+  echo "$output"
+  [ "$status" -eq 0 ]
+  [ "$output" = '["RUN touch /onbuild1" "RUN touch /onbuild2"]' ]
+  cid=$(buildah from ${target})
+  root=$(buildah mount ${cid})
+  run ls ${root}/onbuild1 ${root}/onbuild2
+  echo "$output"
+  [ "$status" -eq 0 ]
+  buildah umount ${cid}
+  buildah rm ${cid}
+
+  target=onbuild-image2
+  buildah bud --format docker --signature-policy ${TESTSDIR}/policy.json -t ${target} -f Dockerfile1 ${TESTSDIR}/bud/onbuild
+  run buildah --debug=false inspect --format '{{printf "%q" .Docker.Config.OnBuild}}' ${target}
+  echo "$output"
+  [ "$status" -eq 0 ]
+  [ "$output" = '["RUN touch /onbuild3"]' ]
+  cid=$(buildah from ${target})
+  root=$(buildah mount ${cid})
+  run ls ${root}/onbuild1 ${root}/onbuild2
+  echo "$output"
+  [ "$status" -eq 0 ]
+  run ls ${root}/onbuild3
+  echo "$output"
+  [ "$status" -ne 0 ]
+  buildah umount ${cid}
+
+  target=onbuild-image3
+  run buildah --debug=false config --onbuild "RUN touch /onbuild4" ${cid}
+  echo "$output"
+  [ "$status" -eq 0 ]
+
+  buildah commit --signature-policy ${TESTSDIR}/policy.json --format docker ${cid} ${target}
+  run buildah --debug=false inspect --format '{{printf "%q" .Docker.Config.OnBuild}}' ${target}
+  echo "$output"
+  [ "$status" -eq 0 ]
+  [ "$output" = '["RUN touch /onbuild3" "RUN touch /onbuild4"]' ]
```

Comment:
Should the output be just "RUN touch /onbuidl4" here?  I thought that's what the clearonbuild function did.